newIDE/app/src/MainFrame/EditorContainers/BaseEditor.js

```diff
@@ -91,6 +91,7 @@ export type RenderEditorContainerProps = {|
     newName: string,
     cb: (boolean) => void
   ) => void,
+  canInstallPrivateAsset: () => Promise<boolean>,
```

Comment:
Should it go to `extraEditorProps`?

Reply:
I'm not sure about the purpose of extraEditorProps.
It looks like it's mainly for optional props, but this one is actually mandatory

Reply:
I understand it differently than the comment: for me it's the space where we put the props that are only used by a single editor in order to avoid putting something only used by the external layout editor in the "base" props